packages/cadl-ranch-specs/http/azure/core/traits/mockapi.ts

```diff
@@ -17,6 +17,14 @@ Scenarios.Azure_Core_Traits_smokeTest = passOnSuccess(
     if (req.params.id !== "1") {
       throw new ValidationError("Expected path param id=1", "1", req.params.id);
     }
+
+    if (!("repeatability-request-id" in req.headers)) {
+      throw new ValidationError("Repeatability-Request-ID is missing", "A UUID string", undefined);
+    }
+    if (!("repeatability-first-sent" in req.headers)) {
```

Comment:
Do we also want to validate the format and not just check for the presence of the header?

Reply:
We could, my JS skills stopped here for this first PR :)

Reply:
It would be great to validate the format as we don't want different languages using different datetime format for this header.

Reply:
We would need to find a JS parser for that, I'm sure that exists :)